modules/cli/pages/index.adoc

```diff
@@ -348,15 +348,29 @@ $ skupper service bind hello-world-backend deployment hello-world-backend
 
 This section shows how to expose a service from a namespace where {skupper-name} is not deployed.
 
+{skupper-name} allows you expose services from other namespaces by default. 
```

Comment:
This sentence seems wrong to me. The following sentence makes more sense.

Reply:
Tried to address this and other comment, maybe easier to review this preview?
![image](https://github.com/skupperproject/skupper-docs/assets/5154224/ce391681-2526-4b28-b3af-18b5f6d8c118)